proto/invocation.proto

```diff
@@ -176,6 +176,25 @@ message InvocationQuery {
 
   // The timestamp up to which the build was last updated (exclusive).
   google.protobuf.Timestamp updated_before = 8;
+
+  // Allowed invocation statuses.
+  repeated StatusFilterValue statuses = 9;
+}
+
+enum StatusFilterValue {
```

Comment:
Also considered `CompositeStatus` and `ClientStatus` but not sure if those are very intuitive -- open to other suggestions

Reply:
OverallStatus? InvocationStatus is taken I guess? Maybe @siggisim has ideas

Reply:
`OverallStatus` SG, if that works for Siggi :)